examples/models/llama2/export_llama_lib.py

```diff
@@ -128,6 +129,10 @@ def build_args_parser() -> argparse.ArgumentParser:
             "qnn_8a8w",
             "qnn_16a16w",
             "qnn_16a4w",
+            "coreml",
+            "coreml_qc4",
+            "coreml_xnnpack",
+            "coreml_xnnpack_qc4",
```

Comment:
Are you using xnnpack quantizer for some specific feature? It's probably better to mark coreml_featurex, otherwise, we may thought it's using both xnnpack and coreml.

Reply:
In this initial PR, we use xnnpack quantizer to quantize the model, then delegate to coreml backend. This serves as the baseline

Rename these to `coreml_baseline`